test/testdata/ppd-valid.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
I generated this file from `cmd/readACH` via https://github.com/moov-io/ach/pull/293